src/exporter/trace/stdout/mod.rs

```diff
@@ -0,0 +1,49 @@
+use crate::exporter::trace;
+use std::any;
+use std::sync::Arc;
+
+#[derive(Debug)]
+/// Exporter
+pub struct Exporter {}
+
+#[derive(Debug)]
+/// Builder
+pub struct Builder {}
+
+impl Default for Builder {
+    /// Return the default Exporter Builder.
+    fn default() -> Self {
+        Builder {}
+    }
+}
+
+impl Builder {
+    /// Create a new exporter from the builder
+    pub fn init(self) -> Exporter {
+        Exporter {}
+    }
+}
+
+impl Exporter {
+    /// Create a new exporter builder.
+    pub fn builder() -> Builder {
+        Builder::default()
+    }
+}
+
+impl trace::SpanExporter for Exporter {
+    /// Export spans to stdout
+    fn export(&self, batch: Vec<Arc<trace::SpanData>>) -> trace::ExportResult {
+        println!("{:?}", batch);
+
+        trace::ExportResult::Success
+    }
+
+    /// Ignored for now.
+    fn shutdown(&self) {}
+
+    /// Allows `Exporter` to be downcast from trait object.
```

Comment:
Do we need this?

Reply:
error[E0046]: not all trait items implemented, missing: `as_any` <- is required from the trait

Reply:
error[E0046]: not all trait items implemented, missing: `as_any`, is required from the trait.

Reply:
Thanks - I checked and could see it was part of other implementations.